src/model/station.js

```diff
@@ -0,0 +1,46 @@
+export function getStations() {
+  return JSON.parse(localStorage.getItem('stations'));
+}
+
+export function addStation(newStation) {
+  let oldStations = localStorage.getItem('stations');
+  if (oldStations === null) {
+    localStorage.setItem('stations', JSON.stringify([newStation]));
+    return true;
+  } else {
+    let stations = JSON.parse(oldStations);
+    if (stations.includes(newStation) === false) {
+      stations.push(newStation);
+      localStorage.setItem('stations', JSON.stringify(stations));
+      return true;
+    }
+    return false;
+  }
+}
+
+function checkLineIncludedStation(lines, station) {
+  let ret = lines.every((line) => {
+    if (
+      line.stations.filter((item) => item !== station).length !=
+      line.stations.length
+    ) {
+      return false;
+    } else {
+      return true;
+    }
+  });
+  return ret;
+}
```

Comment:
```suggestion
function checkLineIncludedStation(lines, station) {
  return lines.every((line) => line.stations.filter((item) => item !== station).length === line.stations.length)
}
```

다음 코드가 가독성 측면에서 조금 더 좋은 것 같아서 리뷰했습니다.

Reply:
와우 엄청나네요